README.md

```diff
@@ -121,3 +157,49 @@ spec:
     aws:
       instanceType: m5.large
 ```
+## Trouble Shooting
+
+**Pull Secret Issues**
+
+- If you run into an issue where the `pods` are not creating properly when you 
+issue the `hypershift create cluster ...` command, it may be your `pull-secret`. There may be a 
+typo or trouble finding the file. Make sure the `pull-secret` is accurate and exists on your system and you 
```

Comment:
If there's any trouble finding the file our cli should fail. Is it not the case?

Reply:
yep, you are correct, I will reword that